__tests__/app.test.js

```diff
@@ -364,6 +364,40 @@ describe("ALL categories of undefined path", () => {
   });
 });
 
+describe("GET /api/users", () => {
+
+  test("status 200: responds with an array of objects with properties of slug and description", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toHaveLength(4);
+
+        users.forEach((user) => {
+          expect(user).toEqual(
+            expect.objectContaining({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
+  test("status 404: Invalid path", () => {
```

Comment:
Do we need this test? Have you previously checked for invalid path anywhere else? This shouldn't be a unique test for this endpoint so shouldn't be in this desc block.